collaborative-filter.Rmd

```diff
@@ -77,7 +79,7 @@ combined_matrix <- cbind(tc_pca[["x"]][1:22,1],
                          tc_pca[["x"]][111:132,1])
 colnames(combined_matrix) <- c("pred","nlp","sna","neural","viz","loop")
 rownames(combined_matrix) <- i1names
-t_comb_matrix <- t(combined_matrix)
+
  unit.order <- matrix(nrow=6,ncol=6)
```

Comment:
Great work! You have done a really excellent hob putting this together, and in record time. Did you include the functions you were working on somewhere also?

Reply:
I added two new files for the other things you were asking for.  I didn't totally function-ize the recommendation for a student, but I did for the next module bit comparing them to each other.